### PR TITLE
Should not be able to create existing sandbox/container.

### PR DESCRIPTION
**Pull request by Random-Liu**
Add integration test for duplicated sandbox/container creation.

Signed-off-by: Lantao Liu <lantaol@google.com>

### CODE REVIEW

File: integration/duplicate_name_test.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	sb, err := runtimeService.RunPodSandbox(sbConfig)
+	require.NoError(t, err)
+	defer func() {
+		assert.NoError(t, runtimeService.StopPodSandbox(sb))
+		assert.NoError(t, runtimeService.RemovePodSandbox(sb))
+	}()
+
+	t.Logf("Create the sandbox again should fail")
+	_, err = runtimeService.RunPodSandbox(sbConfig)
+	require.Error(t, err)
```
Comment (mikebrow) [inline integration/duplicate_name_test.go]: check for specific error, or is that to brittle?

File: integration/duplicate_name_test.go [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+		"container",
+		pauseImage,
+	)
+	_, err = runtimeService.CreateContainer(sb, cnConfig, sbConfig)
+	require.NoError(t, err)
+
+	t.Logf("Create the container again should fail")
+	_, err = runtimeService.CreateContainer(sb, cnConfig, sbConfig)
+	require.Error(t, err)
+}
```
Comment (mikebrow) [inline integration/duplicate_name_test.go]: test for similar names? or is that to pedantic?

File: integration/duplicate_name_test.go (hunk 1 above)
Comment (Random-Liu) [reply integration/duplicate_name_test.go]: There is no specific error defined in CRI, we could only match error message, which could change overtime. :(

File: integration/duplicate_name_test.go (hunk 2 above)
Comment (Random-Liu) [reply integration/duplicate_name_test.go]: Seems not needed. :p

**Comment by Random-Liu** [comment]
@mikebrow Thanks for reviewing!

**Pull request merged**